app/templates/components/forms/wizard/other-details-step.hbs

```diff
@@ -16,6 +16,7 @@
       <Widgets::Forms::LinkInput
         @hasLinkName={{true}}
         @linkName={{socialLink.name}}
+        @isSocialLink={{true}}
         @inputId={{socialLink.identifier}}
         @segmentedLink={{socialLink.segmentedLink}}
         @isChild={{true}}
```

Comment:
@iamareebjamal how can I change `isSocialLink` to `false`, while creating a new record?

Reply:
I don't understand. While creating a new record? You mean while dynamically adding a new LinkINput?

Reply:
Yes

Reply:
You can check `socialLink.id`

It'll be null for a new record. So, instead of true, you can use that